docs/PROCEDURES.md

```diff
@@ -0,0 +1,69 @@
+# Additional procedures for code maintenance 
+
+## Managing pinned dependencies
+
+The OpenSSF, via the [scorecard](https://securityscorecards.dev/) project recommends that projects pin any
+dependencies they use:
+
+* to ensure reproducibility
+* to reduce the change for rogue dependency updates to compromise software
+
+It's important to note that this requires any changes to dependencies are properly reviews, and
```

Comment:
typo: reviews -> reviewed, right? What (manual steps) does this review entail (beyond usual PR-change-of-code reviews)?

Reply:
The usual with any code change - assess the impact. so if there's an update to a dependency try and test it locally, check the release notes.

Reply:
OK. But the typo still remains.

Reply:
fixed (in next push)